### DISCUSSION TITLE
Can we use different types for `many` options?

### DISCUSSION THREAD

**Discussion by Conaclos** [pacak/bpaf#362] [Q&A] [answered]
Hi! I am one of the core contributor of Biome.

I am looking to add a `many' option.
Is it possible to use another type instead of `Vec`, especially a custom type?
If so, what property must my custom type implement?

**Comment by pacak** [comment, marked as answer]
There's `collect` - it works with anything that is `FromIterator`. https://docs.rs/bpaf/0.9.12/bpaf/trait.Parser.html#method.collect
